leakcanary-android/src/main/res/values-de/leak_canary_strings.xml

```diff
@@ -18,8 +18,12 @@
     <string name="leak_canary_leak_excluded">[Ausgeschlossen] %1$s hat %2$s geleaked</string>
     <string name="leak_canary_analysis_failed">Leak Analyse fehlgeschlagen</string>
     <string name="leak_canary_leak_list_title">Leaks in %s</string>
-    <string name="leak_canary_notification_message">Für mehr Details hier klicken</string>
+    <string name="leak_canary_notification_analysing">Analyzing Heap Dump</string>
     <string name="leak_canary_notification_channel">LeakCanary</string>
+    <string name="leak_canary_notification_dumping">Dumping Heap</string>
+    <string name="leak_canary_notification_foreground_text">LeakCanary is working.</string>
+    <string name="leak_canary_notification_message">Für mehr Details hier klicken</string>
+    <string name="leak_canary_notification_reporting">Reporting LeakCanary result</string>
```

Comment:
cc @Goddchen @K0bin @christxph if one of you is interested in contributing more german translations.